### DISCUSSION TITLE
One Signal Push Notifications

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3417] [Ideas]
If its here or doable please send me in the right direction.

It would be awesome to have an integration to One Signal push notifications. That way you have add subscription base alerts or notifications when dates are coming up, etc.

**Comment by mjashanks** [comment]
Hi There. I'm not familiar with it, but it does have an API: https://onesignal.com/message-api

So, you can possibly use our REST connector to integrate.

**Reply by twilson9034** [reply]
Thats a possible work around i was thinking of. I need to play around with it and see what it will take to make it all work.
